test/run

```diff
@@ -22,6 +27,24 @@ case "${TEST_SCENARIO:=}" in
         RUN_OPTS="$RUN_OPTS "
         PREPARE_OPTS="$PREPARE_OPTS --overlay"
         ;;&
+
+    # split tests into roughly equal scenarios for more parallelism
+    *networking*)
+        RUN_OPTS="$RUN_OPTS $(echo "$ALL_TESTS" | grep -E "$RE_NETWORKING")"
+        PREPARE_OPTS="$PREPARE_OPTS --quick"
+        ;;&
+    *storage*)
+        RUN_OPTS="$RUN_OPTS $(echo "$ALL_TESTS" | grep -E "$RE_STORAGE")"
+        PREPARE_OPTS="$PREPARE_OPTS --quick"
+        ;;&
+    *expensive*)
+        RUN_OPTS="$RUN_OPTS $(echo "$ALL_TESTS" | grep -E "$RE_EXPENSIVE")"
+        PREPARE_OPTS="$PREPARE_OPTS --quick"
+        ;;&
+    *default*)
```

Comment:
we could perhaps call this something more distinctive -- "other"? Let the bikeshedding begin.

Reply:
Yeah, I didn't like "default" either.  "rest"? "other" is nicer, imo.

Reply:
Changed to "other".